src/identity.js

```diff
@@ -1824,7 +1728,7 @@ export default function Identity(mpInstance) {
         identityType,
         newIdentity,
         oldIdentity,
-        newCreatedThisBatch,
+        newCreatedThisBatch, // New Identity Type introduced within the batch
```

Comment:
Rather than having the comment, maybe we can rename this variable to: `wasIdentityCreatedThisBatch`.  I don't really like that either, but it's more informative and supplants the need for a comment.

Reply:
I think my shorthand for booleans is to prefix with `is` so I'm going to go with `isIdentityTypeNewToBatch`